src/rupali/assignment17/ArrayExample2.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Program 2:  return a difference between sum of even number and odd numbers.	
+	From given numbers, count the odd and even numbers.
+	input : 10,23,26,27,88,87,65 
+    output : evenSum - oddSum
+	
+	int getDifference(int[] input){
+	
+	}
+
+ */
+package rupali.assignment17;
+
+import java.util.Scanner;
+
+public class ArrayExample2 {
+	void diffEvenOddsum() {
+		int evensum=0,oddsum=0;
+		Scanner s=new Scanner(System.in);
```

Comment:
We should not ask, user to give inputs in service method,
Ideally we take input from main method then pass those value to methods while calling.